### PR TITLE
added edit page for user profile via employee profile

### PR DESCRIPTION
**Pull request by mzavarka**

### CODE REVIEW

File: apps/schools/domains/common/utils/form.ts [hunk 1]
```diff
@@ -32,10 +32,10 @@ export function isPhoneValid(form: FormInstance, field_name: string) {
...
 
-export function removeEmpty(obj: any): any {
+export function removeEmpty(obj: any, exceptList: string[] = []): any {
     return Object.fromEntries(
         Object.entries(obj)
-            .filter(([_, v]) => v != null && v != '' && v != undefined)
+            .filter(([_, v]) => (v != null && v != '' && v != undefined) || exceptList.includes(_))
             .map(([k, v]) => [k, v === Object(v) ? removeEmpty(v) : v]),
     )
 }
```
Comment (levil664) [inline apps/schools/domains/common/utils/form.ts]:
```
export function removeEmpty(obj: any, exceptList: string[] = []): any {
    return Object.fromEntries(
        Object.entries(obj)
            .filter(([key, value]) => (value != null && value !== '' && value !== undefined) || exceptList.includes(key))
            .map(([key, value]) => [key, value === Object(value) ? removeEmpty(value) : value]),
    )
}

```

File: apps/schools/domains/user/components/profile/profileEdit/index.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                    <Form
+                        form={form}
+                        className={styles.table}
+                        colon={false}
+                        requiredMark={false}
+                        onValuesChange={validationCheck}
+                        onFinish={() => {
+                            handleSubmitForm(user.employee_profile?.id ?? '', form, mutation).then((isSuccess) => {
+                                if (isSuccess) window.location.href = '/user'
+                            })
```
Comment (levil664) [inline apps/schools/domains/user/components/profile/profileEdit/index.tsx]: can we throw a refetch ?

File: apps/schools/domains/user/components/profile/profileEdit/index.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                            <Link href={'/auth/forgot'}>Сбросить пароль</Link>
+                        </Form.Item>
+
+                        <Row className={styles.buttonsContainer}>
+                            <Button
+                                type='schoolDefault'
+                                htmlType='submit'
+                                disabled={!isFormValid}
+                                block
+                                style={{ width: '40%' }}
```
Comment (levil664) [inline apps/schools/domains/user/components/profile/profileEdit/index.tsx]: take it to className

File: apps/schools/domains/user/components/profile/profileEdit/index.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                            >
+                                Сохранить изменения
+                            </Button>
+
+                            <Button
+                                type='schoolDefault'
+                                antdType={'default'}
+                                block
+                                style={{ width: '20%' }}
```
Comment (levil664) [inline apps/schools/domains/user/components/profile/profileEdit/index.tsx]: ^^

File: apps/schools/package.json [hunk 5]
```diff
@@ -8,6 +8,7 @@
     "test:jest": "jest --passWithNoTests",
     "prettier": "prettier --write \"domains/**/*.{ts,tsx}\"",
     "dev": "npm run open-browser && next dev",
+    "row-dev": "next dev",
```
Comment (levil664) [inline apps/schools/package.json]: why do we need this?

Comment (mzavarka) [reply apps/schools/package.json]: renamed it

**Review by levil664 — APPROVED** [review]

**Pull request merged**